### ISSUE TITLE
Function keys F13-F24 not working

### ISSUE THREAD

**Issue by Adam-Vandervorst** [Massdrop/mdloader#12]
Higher keycode keys can't be utilized, HID report descriptor only goes to 101 (Keyboard Application). Not sure if this is the right place.

Related:
https://github.com/qmk/qmk_firmware/issues/4597
Method I used:
https://stackoverflow.com/questions/47126939/how-to-debug-usb-hid-scancode-keycode-translation-in-linux

**Comment by drashna** [comment]
This should be fixed, now.

**Comment by minikold** [comment]
Not sure if this is something on mdloader or something within the configurator on Massdrop itself, assuming that combines with the file making on their qmk_firmware file, but with all up to date files this issue seems to persist. (Not code-savvy, apologies).

I tried to find something following what I saw here: https://github.com/qmk/qmk_firmware/issues/4548, but couldn't find the stuff minimum/maximum related when I was looking through on VS Code. I'm sure I was missing something. Figured I'd just address it here.
